scst-store/amr-graphql-query.hbs.md

```diff
@@ -0,0 +1,255 @@
+# AMR Graphql Querying
+
+This page contains information on how to connect to the graphql playground, and how to query each supported data model.
+See [AMR Data Models](amr-data-model.hbs.md) for more information on the AMR data models.
+
+## Connecting to Graphql Playground
+
+Having ingress enabled is the recommended configuration by VMWare. The SCST-Store and AMR share the same ingress configuration, so setting the Store ingress domain will also set the same for AMR.
+To connect to the AMR Graphql playground when ingress is enabled, go to `https://artifact-metadata-repository.<ingress-domain>/play`.
+On this page users will be able to write and execute their own graphql queries to fetch data from the AMR.
+
+## Querying for Locations
+
+This section provides details on the query parameters accepted for `Location`, and the list of attributes available for `Location`.
+
+### <a id='location-query-params'></a> Location Query Parameters
+
+Below are the supported parameters that users can specify when querying for `Location`, they are all optional.
+- `query`: expects a query object that specifies additional query parameters, supported parameters in this query object are listed below
```

Comment:
Add a blank line between a paragraph and the beginning of a list. VMware docs toolchain will screw up the rendering otherwise.